src/Service/ServiceFactory.php

```diff
@@ -223,4 +223,17 @@ public function getVersionsService(): VersionsService
 
         return $this->instances['VersionsService'];
     }
+
+    public function getClipsService(): ProgrammesAggregationService
+    {
+        if (!isset($this->instances['ClipsService'])) {
```

Comment:
If this is an instance of ProgrammesAggregationService, why is this not called getProgrammesAggregationService?